WalletWasabi.Tests/UnitTests/WabiSabi/Backend/PhaseStepping/StepConnectionConfirmationTests.cs

```diff
@@ -121,12 +128,15 @@ public async Task NotEnoughConfirmedTimedoutDestroysAsync()
 			round.Alices.Add(a3);
 			round.Alices.Add(a4);
 			round.SetPhase(Phase.ConnectionConfirmation);
-			using Arena arena = await WabiSabiFactory.CreateAndStartArenaAsync(cfg, round);
+
+			Prison prison = new();
+			using Arena arena = ArenaBuilder.From(cfg, prison).Create(round);
+			await arena.StartAsync(CancellationToken.None);
```

Comment:
To decide: `ArenaBuilder` has `CreateAndStartAsync` method too but I'm not really sure if it is necessary. I'll modify the PR according to consensus here.